aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -209,6 +209,309 @@ SignalR hub methods now support injecting services through dependency injection
++ <SurveyPrompt />
+```
+
+The change isn't reflected in the UI. This is because the existing `SurveyPrompt` instance already has a value for its `Title` property. The prior value isn't overwritten.
+
+In .NET 7, `SetParametersAsync` determines if a Hot Reload refresh is in progress. If a refresh occurs, `[Parameter]` properties are reset to their default values before receiving the new set of values from the component's parent.
+
+-->
+
+### Dynamic authentication requests with MSAL in Blazor WebAssembly
+
+New in .NET 7, Blazor WebAssembly supports creating dynamic authentication requests at runtime with custom parameters to handle advanced authentication scenarios.
+
+For more information, see the following articles:
+
+* <xref:blazor/security/webassembly/index?view=aspnetcore-7.0#customize-authentication>
+* <xref:blazor/security/webassembly/additional-scenarios?view=aspnetcore-7.0#custom-authentication-request-scenarios>
+
+### Blazor WebAssembly debugging improvements
+
+Blazor WebAssembly debugging has the following improvements:
+
+* Support for the **Just My Code** setting to show or hide type members that aren't from user code.
+* Support for inspecting multidimensional arrays.
+* **Call Stack** now shows the correct name for asynchronous methods.
+* Improved expression evaluation.
+* Correct handling of the `new` keyword on derived members.
+* Support for debugger-related attributes in `System.Diagnostics`.
+
+### `System.Security.Cryptography` support on WebAssembly
+
+.NET 6 supported the SHA family of hashing algorithms when running on WebAssembly. .NET 7 enables more cryptographic algorithms by taking advantage of [:::no-loc text="SubtleCrypto":::](https://developer.mozilla.org/docs/Web/API/SubtleCrypto), when possible, and falling back to a .NET implementation when :::no-loc text="SubtleCrypto"::: can't be used. The following algorithms are supported on WebAssembly in .NET 7:
+
+* SHA1
+* SHA256
+* SHA384
+* SHA512
+* HMACSHA1
+* HMACSHA256
+* HMACSHA384
+* HMACSHA512
+* AES-CBC
+* PBKDF2
+* HKDF
+
+For more information, see [Developers targeting browser-wasm can use Web Crypto APIs (dotnet/runtime #40074)](https://github.com/dotnet/runtime/issues/40074).
+
+<!-- 
+
+    HOLD
+
+    🪲 fix ... not addressed in docs
+
+### Generic Blazor component can't correctly infer it's type when type has an additional constraint
+
+https://github.com/dotnet/aspnetcore/issues/25588
+
+-->
+
+<!--
+
+    HOLD
+
+    🪲 fix ... not addressed in docs
+
+### `CascadingTypeParameter` problem
+
+https://github.com/dotnet/aspnetcore/issues/38479
+
+-->
+
+### Inject services into custom validation attributes
+
+You can now inject services into custom validation attributes. Blazor sets up the `ValidationContext` so that it can be used as a service provider.
+
+For more information, see <xref:blazor/forms-and-input-components?view=aspnetcore-7.0#custom-validation-attributes>.
+
+### Blazor WebAssembly's default browser for debugging adopts Microsoft Edge
```

Comment:
This isn't really a .NET 7 change. I think this might be referring to our VS Code debugging experience for Blazor WebAssembly. It can be removed from this list.